meta-aos-rpi-install/recipes-core/initrdscripts/files/install_script.sh

```diff
@@ -58,32 +58,64 @@ wait_for_block_device() {
     return 0
 }
 
+
+add_user() {
+    mkdir -p /sd1 /flash1
+    mount -t auto /dev/mmcblk0p1 /sd1/
+
+    if [ -f /sd1/firstrun.sh ]; then
```

Comment:
echo "Adding user provided by Raspberry Pi Imager...
```suggestion
    if [ -f /sd1/firstrun.sh ]; then
```